Sofascore2025/LeagueView.swift

```diff
@@ -0,0 +1,101 @@
+import UIKit
+import SnapKit
+import SofaAcademic
+
+class LeagueView: BaseView {
```

Comment:
pogledaj si koja je razlika izmedu final i obicnog class-a ak te zanima
```suggestion
final class LeagueView: BaseView {
```